docs/release-notes/release-notes-0.19.0.md

```diff
@@ -48,6 +49,11 @@
 ## Performance Improvements
 
 * Log rotation can now use ZSTD 
+## Misc
+
+* [Add retry logic for block fetching](https://github.com/lightningnetwork/lnd/pull/8381)
+  block fetching to retry indefinitely with an exponential back-off between
```

Comment:
Thanks for the update, a final nit: 
`block fetching` is repeated here, and `indefinitely` is outdated, maybe you can change it to 

```
* [Add retry logic](https://github.com/lightningnetwork/lnd/pull/8381) for
  watchtower block fetching with a max number of attempts and exponential
  back-off.
```
If you're at it, we sort the list of contributors alphabetically. Also, the commit message is outdated, it mentions three times, but I think we retry five times.